### DISCUSSION TITLE
Using env variable for assetPrefix

### DISCUSSION THREAD

**Discussion by slaviczavik** [vercel/next.js#66607] [Help]
### Summary

Not possible to use environment variable set in Docker during build for `assetPrefix`.

### Additional information

If I use static string for `assetPrefix` everything works as expected, but if I use env variable set in Docker before build, some assets have wrong URL and all asset URI throws 404.

But if the same env variable is set in the `.env` file everything again works as expected.

Is there any documentation why is this happening?

**Comment by SouthLink** [comment]
I think you can use run time [environment variables,](https://nextjs.org/docs/app/building-your-application/configuring/environment-variables#runtime-environment-variables) you can support build time and run time environment variables

**Comment by rohitpotato** [comment]
Did you manage to find a solution for this? @slaviczavik
